### PR TITLE
Changed Shortcut icons will appear "round"  to "smaller" 

### PR DESCRIPTION
**Pull request by prem-k-r**
Same in Hindi and Bengali

### CODE REVIEW

**Comment by prem-k-r** [comment]
> I know you used the Google Translator but it has some issues it always shows the wrong translation. So do the changes.

Nah, I didn't. 
Both are correct, but 'করুন' gives a clearer instruction, while 'হবে' makes it sound more passive. I followed formal and action-oriented tone as in rest of entities. 

I'll do the change as you asked.

File: languages.js [hunk 1]
```diff
@@ -572,7 +572,7 @@ const translations = {
         "editShortcutsList": "সংরক্ষিত শর্টকাট",
         "shortcutsInfoText": "যেসব শর্টকাট প্রদর্শিত হবে তা নির্বাচন করুন",
         "adaptiveIconText": "অ্যাডাপ্টিভ আইকন আকার",
-        "adaptiveIconInfoText": "শর্টকাট আইকনগুলি গোলাকৃতি প্রদর্শিত হবে",
+        "adaptiveIconInfoText": "শর্টকাট আইকন ছোট আকারে প্রদর্শন করুন",
```
[the remark this answers is not in the document]
Comment (prem-k-r) [reply languages.js]: Done

**Review by itz-rj-here — APPROVED** [review]
Great thanks

**Comment by prem-k-r** [comment]
> I know you used the Google Translator but it has some issues it always shows the wrong translation. So do the changes.

IMG_20241107_190000
Google translator lol 😂

**Comment by itz-rj-here** [comment]
> > I know you used the Google Translator but it has some issues it always shows the wrong translation. So do the changes.
> 
> IMG_20241107_190000 Google translator lol 😂

Bruh -_- forget that.

**Comment by ceskyDJ** [comment]
Could you please update the Czech translation?
```diff
-"adaptiveIconInfoText": "Ikony zkratek se zakulatí",
+"adaptiveIconInfoText": "Ikony zkratek se zmenší",
```

**Pull request merged**